components/ntp_background_images/browser/ntp_background_images_service.cc

```diff
@@ -44,7 +44,8 @@ namespace ntp_background_images {
 
 namespace {
 
-constexpr int kSIComponentUpdateCheckIntervalMins = 15;
+constexpr base::TimeDelta kSIComponentUpdateCheckIntervalMins =
```

Comment:
nit: Can we remove `Mins` because `base::Seconds`, `Minutes` etc. would determine this.

Reply:
Fixed